unified-scheduler-pool/src/lib.rs

```diff
@@ -296,6 +296,8 @@ impl BankingStageHelper {
 
     pub fn send_new_task(&self, task: Task) {
         self.new_task_sender
+            .upgrade()
+            .unwrap()
```

Comment:
should we instead just drop the task if the upgrade fails?

Reply:
I preserved the current behavior of panicing on failure to send; realistically we probably want to return an error and break whatever loops we're in wherever we send these

Reply:
yeah, I'm okay w/ following up on that separately